initLocalTest.sh

```diff
@@ -165,6 +165,8 @@ helmUpgrade galoy \
   $INFRADIR/galoy/
 
 kubectlWait app.kubernetes.io/instance=galoy
+kubectlWait statefulset.kubernetes.io/pod-name=galoy-redis-node-1
+kubectlWait statefulset.kubernetes.io/pod-name=galoy-redis-node-2
```

Comment:
we should wait for 3 nodes and just 2?

Reply:
Actually these two lines aren't needed because of the line above them which is waiting for instance=galoy. I've removed it.